src/lsm/lsm_worker.c

```diff
@@ -20,7 +20,7 @@ int
 __wt_lsm_worker_start(WT_SESSION_IMPL *session, WT_LSM_WORKER_ARGS *args)
 {
 	WT_RET(__wt_verbose(session, WT_VERB_LSM_MANAGER,
-	    "Start LSM worker %d type 0x%x", args->id, args->type));
+	    "Start LSM worker %u type %#" PRIx32, args->id, args->type));
```

Comment:
Again not sure why `wt_lsm_worker_args->id` is a `u_int` instead of a sized `uint32_t` or `uint64_t`.  I guess it is beyond the scope of this ticket, but should we revisit those too?  In for a penny in for a pound...